src/crate/theme/rtd/conf/__init__.py

```diff
@@ -30,13 +30,25 @@
 exclude_trees = ["pyenv", "tmp", "out", "parts", "clients", "eggs"]
 
 extensions = [
+    "myst_parser",
+    "sphinx_copybutton",
+    "sphinx_design",
+    "sphinx_inline_tabs",
     "sphinx_sitemap",
+    "sphinx_subfigure",
+    "sphinx_togglebutton",
     "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinx.ext.ifconfig",
+    "sphinx.ext.graphviz",
+    "sphinxcontrib.mermaid",
     "sphinxcontrib.plantuml",
     "sphinxext.opengraph",
-    "sphinx_copybutton",
```

Comment:
> I noticed you suggest removal of sphinx_copybutton.
>> That would not have been intended. Please add an inline comment on the PR where you discovered this spot.

Right here, @amotl

Reply:
Yeah. List is sorted alphabetically now. ;]